translations/base-de.yaml

```diff
@@ -663,48 +663,45 @@ storyRewards:
         desc: Du hast eine weitere Variante des <strong>Rotierers</strong> freigeschaltet! Mit ihm
             kannst du Formen um 180° drehen (Überraschung! :D).
     reward_wires_painter_and_levers:
-        title: Wires & Quad Painter
-        desc: "You just unlocked the <strong>Wires Layer</strong>: It is a separate
-            layer on top of the regular layer and introduces a lot of new
-            mechanics!<br><br> For the beginning I unlocked you the <strong>Quad
-            Painter</strong> - Connect the slots you would like to paint with on
-            the wires layer!<br><br> To switch to the wires layer, press
-            <strong>E</strong>."
+        title: Kabel & Färber (4-fach)
+        desc: "Du hast die <strong>Kabelebene</strong> freigeschaltet: Diese ist eine
+            seperate Ebene über der regulären Ebene und kommt mit vielen neuen Mechaniken!
+            <br><br> Für den Anfang habe ich dir den <strong>Quad Färber</strong>
+            freigeschaltet - Verbinde die Slots mit denen du malen willst mit der Kabelebene!<br><br>
```

Comment:
`die Slots, mit denen du malen willst, mit` Interpunktion. `möchtest` klingt freundlicher

Reply:
Ich habe in der Übersetzung fast alle Bindestriche durch Satzzeichen ersetzt, also auch hier bitte durch ein Ausrufezeichen ersetzen und den folgenden Satz mit einem Punkt beenden